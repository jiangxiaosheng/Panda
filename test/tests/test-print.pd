func main() {
    var v = "hello world"
    print(v)
}