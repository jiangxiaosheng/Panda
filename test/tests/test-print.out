

Sementically checked program: 


void main() {
var v: string(string : hello world)
(int : print((string : v)))
}

